### PR TITLE
Add Release Configuration

### PR DESCRIPTION
**Pull request by Nitish145**

### CODE REVIEW

File: lib/config/environment_config.dart [hunk 1]
```diff
@@ -3,4 +3,18 @@ class EnvironmentConfig {
...
   );
+
+  // GITHUB OAUTH ENV VARIABLES
+  static const String GITHUB_OAUTH_CLIENT_ID = String.fromEnvironment(
+    'GITHUB_OAUTH_CLIENT_ID',
+    defaultValue: '06fa6b965ba7e4a4175c',
+  );
+  static const String GITHUB_OAUTH_CLIENT_SECRET = String.fromEnvironment(
+    'GITHUB_OAUTH_CLIENT_SECRET',
+    defaultValue: 'd3a8a95bd1691b557ca347b14ae6507b89f92e47',
```
Comment (tachyons) [inline lib/config/environment_config.dart:14]: Revoke this secret

**Comment by Nitish145** [comment]
@tachyons please check!

File: lib/config/environment_config.dart (hunk 1 above)
Comment (Nitish145) [reply lib/config/environment_config.dart]: @tachyons please check!

File: ios/Runner/Info.plist [hunk 2]
```diff
@@ -53,14 +53,14 @@
...
-                <string>fb567334037481721</string>
+                <string>FB_LOGIN_PROTOCOL_SCHEME</string>
             </array>
         </dict>
 		<!-- Google authentication config -->
 		<dict>
 			<key>CFBundleURLSchemes</key>
 			<array>
-                <string>com.googleusercontent.apps.377832615490-d5i0e2guo62fja096ugpnjak12sleo8m</string>
+                <string>REVERSED_CLIENT_ID</string>
```
Comment (tachyons) [inline ios/Runner/Info.plist:63]: REVERSED ?

Comment (Nitish145) [reply ios/Runner/Info.plist:63]: This is the way cliend_id is present in google services file with this technical name. @tachyons

**Pull request merged**